### PR TITLE
MarkDown update: replace deprecated HTML tags

### LINKED ISSUE #9449: <strong> tags causing issue in localized articles
The following English articles contain <strong> tags which are not valid .md syntax.  
https://docs.microsoft.com/en-us/windows/client-management/mdm/diagnosticlog-csp
https://docs.microsoft.com/en-us/windows/client-management/mdm/provisioning-csp
https://docs.microsoft.com/en-us/windows/client-management/mdm/proxy-csp
https://docs.microsoft.com/en-us/windows/client-management/mdm/win32appinventory-csp
https://docs.microsoft.com/en-us/windows/client-management/mdm/update-csp

When the articles are localized the pages appear messed up because of the <strong> tags
https://docs.microsoft.com/ja-jp/windows/client-management/mdm/diagnosticlog-csp
https://docs.microsoft.com/ja-jp/windows/client-management/mdm/provisioning-csp
https://docs.microsoft.com/ja-jp/windows/client-management/mdm/proxy-csp
https://docs.microsoft.com/ja-jp/windows/client-management/mdm/win32appinventory-csp
https://docs.microsoft.com/ja-jp/windows/client-management/mdm/update-csp
image

Can you please fix the English topics
See the related issue: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9426
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a84b842f-150d-f006-0563-db5a0f01a64f
* Version Independent ID: 7dd55cbe-abc2-9c33-3fef-843b87f26312
* Content: [DiagnosticLog CSP - Windows Client Management](https://docs.microsoft.com/en-us/windows/client-management/mdm/diagnosticlog-csp)
* Content Source: [windows/client-management/mdm/diagnosticlog-csp.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/client-management/mdm/diagnosticlog-csp.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @ManikaDhiman
* Microsoft Alias: **dansimp**

### PR DESCRIPTION
**Pull request by illfated**
**Description:**

As requested in issue ticket #9449 (**\<strong\> tags causing issue in localized articles**), this Pull Request aims to remove HTML tags known to cause layout and readability issues in the MS Docs document pages, Frame Overflow being a frequent and obvious layout-breaking issue in translated articles.

Thanks to @TinaMcN for pointing out the issue itself and the most requested articles in need of correction to resolve this issue.

**Changes proposed:**

- Replace HTML tag \<strong\> \</strong\> with its MD equivalent ** **
- Replace single asterisk * with underscore _ (when combined with **)
- Simplify usage of bold formatting (** **) to only the outer pair
- Replace HTML tag \<em\> \</em\> with MarkDown underscore _ _

**Whitespace changes:**

- Remove redundant end-of-line (EOL) blanks (trailing space)
- Normalize spacing in numbered lists and bullet point lists to 1 space
- Replace end-of-file (EOF) consecutive blank lines with 1 NewLine
- Normalize layout spacing in MarkDown tables (consistency)
- Normalize layout to 1 blank line after H2, H3, H4 headings
- Allow up to 2 blank lines spacing (not 3 or more), ex. H1 page title

**Additional notes:**

There may be some more improvements in these articles than described above, both from the initial PR edits as well as from copy review (ideally performed by @JohanFreelancer9 and any of the MS Docs team members) during the review time of this PR. Feel free to suggest further improvements, preferably using the MS Docs conventions & codestyle.

**Ticket closure or reference:**

Closes #9449

### CODE REVIEW

**Review by illfated — COMMENTED** [review]
**Regular source file view:**
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9452/files

**Hide whitespace changes** to simplify the review process by skipping lines where only end-of-line (EOL) blanks have been removed:
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9452/files?diff=unified&w=1

Due to the amount of whitespace changes and there being 5 separate files included in this PR, I suggest using the Rich Diff view at your own discretion, by clicking the individual file buttons in the PR file view.

**Comment by illfated** [comment]
#sign-off (for now)

File: windows/client-management/mdm/provisioning-csp.md [hunk 1]
```diff
@@ -40,43 +40,34 @@ ProvisioningCommands
...
+<a href="" id="--vendor-msft"></a>**./Vendor/MSFT/ProvisioningCommands**
 Root node for Provisioning CSP.
 
-<a href="" id="provisioning-enrollments"></a>**Provisioning/Enrollments**  
+<a href="" id="provisioning-enrollments"></a>**Provisioning/Enrollments**
 Node for defining bulk enrollment of users into an MDM service.
 
-<a href="" id="provisioning-enrollments-upn"></a>**Provisioning/Enrollments/**<strong>*UPN*</strong>  
+<a href="" id="provisioning-enrollments-upn"></a>**Provisioning/Enrollments/_UPN_**
 Unique identifier for the enrollment. For bulk enrollment, this must a service account that is allowed to enroll multiple users. Example, "generic-device@contoso.com"
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/provisioning-csp.md:51]:
```suggestion
Unique identifier for the enrollment. For bulk enrollment, this must be a service account that is allowed to enroll multiple users. For example, "generic-device@contoso.com".
```

File: windows/client-management/mdm/provisioning-csp.md [hunk 2]
```diff
@@ -40,43 +40,34 @@ ProvisioningCommands
...
-<a href="" id="provisioning-enrollments-upn-secret"></a>**Provisioning/Enrollments/*UPN*/Secret**  
+<a href="" id="provisioning-enrollments-upn-secret"></a>**Provisioning/Enrollments/*UPN*/Secret**
 This information is dependent on the AuthPolicy being used. Possible values:
 
--   Password string for on-premises authentication enrollment
--   Federated security token for federated enrollment
--   Certificate thumb print for certificated based enrollment
+- Password string for on-premises authentication enrollment
+- Federated security token for federated enrollment
+- Certificate thumb print for certificated based enrollment
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/provisioning-csp.md:61]:
```suggestion
- Certificate thumbprint for certificated based enrollment
```

File: windows/client-management/mdm/proxy-csp.md [hunk 3]
```diff
@@ -56,82 +55,73 @@ Proxy
...
+<a href="" id="--vendor-msft-proxy"></a>**./Vendor/MSFT/Proxy**
 Root node for the proxy connection.
 
-<a href="" id="proxyname"></a>***ProxyName***  
+<a href="" id="proxyname"></a>***ProxyName***
 Defines the name of a proxy connection.
 
 It is recommended that this element name is specified as a numbered node beginning at zero. For example, to provision two proxy connections, use "PROXY0" and "PROXY1" as the element names. Any unique name can be used if desired (such as "GPRS-NAP"), but no spaces may appear in the name (use %20 instead).
 
 The addition, update, and deletion of this sub-tree of nodes have be specified in a single atomic transaction.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/proxy-csp.md:67]:
```suggestion
The addition, update, and deletion of this sub-tree of nodes have to be specified in a single atomic transaction.
```

File: windows/client-management/mdm/update-csp.md [hunk 4]
```diff
@@ -70,7 +71,7 @@ Update
 
 <p style="margin-left: 20px">Supported operations are Get and Add.
 
-<a href="" id="approvedupdates-approved-update-guid"></a>**ApprovedUpdates/**<strong>*Approved Update Guid*</strong>
+<a href="" id="approvedupdates-approved-update-guid"></a>**ApprovedUpdates/_Approved Update Guid_**
 <p style="margin-left: 20px">Specifies the update GUID.
 
 <p style="margin-left: 20px">To auto-approve a class of updates, you can specify the <a href="/previous-versions/windows/desktop/ff357803(v=vs.85)" data-raw-source="[Update Classifications](/previous-versions/windows/desktop/ff357803(v=vs.85))">Update Classifications</a> GUIDs. We strongly recommend to always specify the DefinitionsUpdates classification (E0789628-CE08-4437-BE74-2495B842F43B), which are used for anti-malware signatures. There are released periodically (several times a day). Some businesses may also want to auto-approve security updates to get them deployed quickly.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/update-csp.md:77]:
```suggestion
<p style="margin-left: 20px">To auto-approve a class of updates, you can specify the <a href="/previous-versions/windows/desktop/ff357803(v=vs.85)" data-raw-source="[Update Classifications](/previous-versions/windows/desktop/ff357803(v=vs.85))">Update Classifications</a> GUIDs. We strongly recommend to always specify the DefinitionsUpdates classification (E0789628-CE08-4437-BE74-2495B842F43B), which are used for anti-malware signatures. These are released periodically (several times a day). Some businesses may also want to auto-approve security updates to get them deployed quickly.
```

File: windows/client-management/mdm/update-csp.md [hunk 5]
```diff
@@ -93,7 +94,7 @@ Update
 
 <p style="margin-left: 20px">Supported operation is Get.
 
-<a href="" id="failedupdates-failed-update-guid"></a>**FailedUpdates/**<strong>*Failed Update Guid*</strong>
+<a href="" id="failedupdates-failed-update-guid"></a>**FailedUpdates/_Failed Update Guid_**
 <p style="margin-left: 20px">Update identifier field of the UpdateIdentity GUID that represent an update that failed to download or install.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/update-csp.md:98]:
```suggestion
<p style="margin-left: 20px">Update identifier field of the UpdateIdentity GUID that represents an update that failed to download or install.
```

File: windows/client-management/mdm/diagnosticlog-csp.md [hunk 6]
```diff
@@ -1418,26 +1421,28 @@ The supported value is Execute.
...
+<a href="" id="filedownload"></a>**FileDownload**
 Node to contain child nodes for log file transportation protocols and corresponding actions.
 
-<a href="" id="filedownload-dmchannel"></a>**FileDownload/DMChannel**  
+<a href="" id="filedownload-dmchannel"></a>**FileDownload/DMChannel**
 Node to contain child nodes using DM channel for transport protocol.
 
-<a href="" id="filedownload-dmchannel-filecontext"></a>**FileDownload/DMChannel/**<strong>*FileContext*</strong>  
+<a href="" id="filedownload-dmchannel-filecontext"></a>**FileDownload/DMChannel/_FileContext_**
 Dynamic interior nodes that represents per log file context.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/diagnosticlog-csp.md:1443]:
```suggestion
Dynamic interior nodes that represent per log file context.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: windows/client-management/mdm/provisioning-csp.md (hunk 1 above)
Comment (illfated) [reply windows/client-management/mdm/provisioning-csp.md:51]: Good point, thanks for pointing it out.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Comment by illfated** [comment]
Thank you for the follow-up, both of you. 👍🏼

**Pull request merged**